frontend/src/App.jsx

```diff
@@ -1,31 +1,23 @@
-import { createBrowserRouter, RouterProvider } from "react-router-dom";
+import { BrowserRouter, Route, Routes } from "react-router-dom";
 
 import "./App.css";
 import { HomePage } from "./pages/Home/HomePage";
 import { LoginPage } from "./pages/Login/LoginPage";
 import { SignupPage } from "./pages/Signup/SignupPage";
+import Navbar from "./components/Navbar";
 
 // docs: https://reactrouter.com/en/main/start/overview
-const router = createBrowserRouter([
-  {
-    path: "/",
-    element: <HomePage />,
-  },
-  {
-    path: "/login",
-    element: <LoginPage />,
-  },
-  {
-    path: "/signup",
-    element: <SignupPage />,
-  },
-]);
 
 const App = () => {
   return (
-    <>
-      <RouterProvider router={router} />
-    </>
+    <BrowserRouter>
```

Comment:
What was the reason for the changes?
Just want to understand the difference between your code and the previous.

Reply:
Originally it was because I have used BrowserRouter before and not CreateBrowserRouter - it seems you can use one or the other. Also when I wrote it I thought that CreateBrowserRouter was available only on an older version of react-router that we hadn't installed and so It would be simpler to use BrowserRouter.

Reply:
Ahh, okok. So it's just different way of doing the same thing?

Reply:
In hindsight I would have used the recommended way with CreateBrowserRouter